great-idea/css/index.css

```diff
@@ -16,13 +20,16 @@ article, aside, canvas, details, embed,
 figure, figcaption, footer, header, hgroup, 
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
-	margin: 0;
-	padding: 0;
+	margin: 20;
```

Comment:
Adding properties to a bunch of elements/classes at once is really efficient and it's a good coding practice when there's a use for it,, but I do want to caution you to think carefully about inheritance when you assign them like this. Many properties are inherited, but some are not -- and if the code you're writing applies to nearly _every_ element on the page, it can be hard to predict and troubleshoot the changes you're making.